--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,6 @@
 NAME=wvkbd
 BIN=${NAME}-${LAYOUT}
 SRC=.
-MAN1 = ${NAME}.1
 
 PKGS = wayland-client xkbcommon pangocairo
 
@@ -25,11 +24,7 @@
 SOURCES = $(WVKBD_SOURCES) $(WAYLAND_SRC)
 OBJECTS = $(WVKBD_DIR_SOURCES:.c=.o) $(WAYLAND_SRC:.c=.o)
 
-SCDOC=scdoc
-DOCS = wvkbd.1
-
-
-all: ${BIN} ${DOCS}
+all: ${BIN}
 
 $(BUILDDIR)/config.h:
 	mkdir -p $(BUILDDIR)
@@ -56,13 +51,7 @@
 format:
 	clang-format -i $(WVKBD_SOURCES) $(WVKBD_HEADERS)
 
-%: %.scd
-	$(SCDOC) < $< > $@
-
 install: all
 	mkdir -p ${DESTDIR}${PREFIX}/bin
 	cp -f ${NAME}-${LAYOUT} ${DESTDIR}${PREFIX}/bin
 	chmod 755 ${DESTDIR}${PREFIX}/bin/${NAME}-${LAYOUT}
-	mkdir -p "${DESTDIR}${MANPREFIX}/man1"
-	sed "s/VERSION/${VERSION}/g" < ${MAN1} > ${DESTDIR}${MANPREFIX}/man1/${MAN1}
-	chmod 644 ${DESTDIR}${MANPREFIX}/man1/${MAN1}
